src/client/src/rt-platforms/glue/glueCore.ts

```diff
@@ -0,0 +1,91 @@
+import ReactGA from 'react-ga'
+import GlueWeb from '@glue42/web'
+import { WindowConfig } from '../types'
+import { Platform } from '../platform'
+import { registerWindowMethods } from './window'
+import { sendNotification, NotifyPermission } from './utils/sendNotification'
+import { ApplicationEpic } from 'StoreTypes'
+import { GlueHeader, GlueLogoLink, GlueCoreRoute } from './'
+
+
+export class Glue42Core implements Platform {
+  allowTearOff = true
+  allowPopIn = true
+  readonly name = 'glue-core'
+  readonly type = 'browser'
+  style = {
+    height: '100%',
+  }
+
+  constructor() {
+    GlueWeb()
+      .then(glue => {
+        window.glue = glue;
+        registerWindowMethods()
+      })
+      .catch((e) => {
+        console.log(e)
+        throw new Error('Failed to init Glue42Web')
+      })
+  }
+
+  window = {
+    close: () => {
+      ReactGA.event({
+        category: 'RT - Window',
+        action: 'close',
+        label: window.name,
+      })
+      return Promise.resolve(window.close())
+    },
+
+    open: (config: WindowConfig, onClose?: () => void) => {
+      ReactGA.event({
+        category: 'RT - Window',
+        action: 'open',
+        label: config.name,
+      })
+      window.glue.windows.onWindowRemoved((windowRemoved: any) => {
+        onClose && onClose()
+      })
+
+      const paramType = config.url.includes('?') ? '&' : '?';
+      const url = `${config.url}${paramType}glue=CORE`;
+
+      return window.glue.windows
+        .open(config.name, url, config)
+        .then((createdWindow: any) => createdWindow)
+    },
+
+    /**
+     * In order to integrate Glue42 with channels the clicked symbol needs to be published to the channel.
+     */
+    publishToChannel: (symbol: string) => {
+      try {
+        window.glue.channels.publish({ symbol: symbol.replace('/', '') })
+      } catch (err) {
+        console.warn(err.message)
+      }
+    },
+  }
+
+  notification = {
+    notify: (message: object) => {
+      if ('Notification' in window) {
+        if (Notification.permission === NotifyPermission.granted) {
+          sendNotification(message)
+        }
```

Comment:
How about the notification, does it actually work, or shall we remove it for now as well?